suites/reef/rados/tier-3_rados_cidr_blocklisting.yaml

```diff
@@ -166,3 +166,9 @@ tests:
             conf: sample-pool-2
         pool_configs_path: "conf/pacific/rados/test-confs/pool-configurations.yaml"
       desc: CIDR Blocklisting of ceph rbd clients
+
+  - test:
```

Comment:
Would the BZ be backported to pacific? Can we add the test in suite and then comment it os that we have it as backlog item?

Reply:
Not sure when and if it will be backported, I have added the test in Pacific test suite as well